File: oshi-front/src/apis/request/goods/UpdateGoods.Request.Dto.ts
export default interface UpdateGoodsRequestDto {
    goodsId: number; // @NotNull -> number 타입
    name: string; // @NotBlank -> string 타입
    description?: string; // Optional field
    type: number; // @NotNull -> number 타입
    file?: File[]; // MultipartFile 리스트 -> File 배열로 변환
  }

File: ohsi-back/src/main/java/com/oshi/ohsi_back/dto/request/goods/UpdateGoodsRequestDto.java
package com.oshi.ohsi_back.dto.request.goods;


import java.util.List;

import org.springframework.web.multipart.MultipartFile;

import io.micrometer.common.lang.Nullable;
import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class UpdateGoodsRequestDto {
    
    @NotNull
    private int goodsId;
    @NotBlank
    private String name;
    @Nullable
    private String description;
    @NotNull
    private int type;

    List<MultipartFile> file;
    
}
